source/man/doveadm-stats.1.rst.in

```diff
@@ -35,6 +37,8 @@ DESCRIPTION
 
 **doveadm stats remove** is used to remove metrics from statistics
 
+**doveadm stats reopen** is used to reopen any file exporter files.
```

Comment:
I think it reopens also all unix socket exporters? I wonder if it should be doing that. But at least if it does, should say it here. Also same in the other commit.

Reply:
Updated code to match documentation